tutorials/server/ubuntu-desktop-aws/ubuntu-desktop-aws.md

```diff
@@ -0,0 +1,153 @@
+---
+id: tutorial-ubuntu-desktop-aws
+summary: Learn how to connect and run ubuntu desktop on AWS EC2 Instance. Access GUI to manage your machine more easily.
+categories: server
+tags: vnc,aws,ec2,tutorial,ubuntu,server,Ubuntu 16.04 LTS
+difficulty: 2
+status: published
+published: 2017-12-16
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+author: Simran Singh <invincible.simran@gmail.com>
+
+---
+
+# Running Ubuntu Desktop on AWS EC2 Instance
```

Comment:
We would probably put 'an' in there - "... an an AWS EC2"..."

Lower case for 'instance'